frontend/src/initKea.ts

```diff
@@ -83,9 +84,13 @@ export function initKea({ routerHistory, routerLocation, beforePlugins }: InitKe
                     (error?.message === 'Failed to fetch' || // Likely CORS headers errors (i.e. request failing without reaching Django)
                         (error?.status !== undefined && ![200, 201, 204].includes(error.status)))
                 ) {
+                    let errorMessageFallback = 'PostHog may be offline'
+                    if (error.status === 404) {
+                        errorMessageFallback = 'URL not found'
+                    }
                     lemonToast.error(
                         `${identifierToHuman(actionKey)} failed: ${
-                            error.detail || error.statusText || 'PostHog may be offline'
+                            error.detail || error.statusText || errorMessageFallback
```

Comment:
fly-by - when I made the API return 404 I also got a toast saying "posthog may be offline". that doesn't feel like a great thing to say to a user that typed into URL bar or who is in wrong project... so let's not